indico/modules/events/tracks/templates/management.html

```diff
@@ -1,20 +1,25 @@
 {% extends 'layout/full_content_base.html' %}
```

Comment:
Just extend from `base.html`.

```jinja
{% extends 'layout/base.html' %}
```

This will make sure the page is page content is limited to `800px` and you can remove that CSS rule from `#track-list`.